src/rmkit/ui/events.cpy

```diff
@@ -23,14 +23,25 @@ namespace ui:
     KEY_EVENT pressed
   ;
 
+  class WidgetGestureEvent : public input::SynMotionEvent
+    public:
+    bool is_long_press = false;
+    WidgetGestureEvent(const input::SynMotionEvent & ev, bool is_long_press=false) \
+        : input::SynMotionEvent(ev), is_long_press(is_long_press):
+      pass
+  ;
```

Comment:
I chose to go w/ a SynMotionEvent subclass with is_long_down since otherwise I think it means adding three more event handlers (long_press_drag_start, long_press_dragging, long_press_drag_end) which felt like overkill.